webapp/src/sui.tsx

```diff
@@ -52,6 +54,12 @@ function removeClass(el: HTMLElement, cls: string) {
     else if (el.className.indexOf(cls) >= 0) el.className.replace(new RegExp(`(?:^|\\s)${cls}(?:\\s|$)`), ' ');
 }
 
+export function fireClickOnEnter(e: React.KeyboardEvent): void {
+    if (e.charCode == 13) {
```

Comment:
Can we handle Space as well? 
Space = 32

Not all browsers store the character value in charCode. use .which instead. 

let charCode = (typeof e.which == "number") ? e.which : e.keyCode